controllers/velero.go

```diff
@@ -355,19 +355,13 @@ func (r *VeleroReconciler) buildVeleroDeployment(veleroDeployment *appsv1.Deploy
 	for _, plugin := range velero.Spec.DefaultVeleroPlugins {
 		if plugin == oadpv1alpha1.DefaultPluginCSI {
 			// CSI plugin is added so ensure that CSI feature flags is set
-			foundCSIFeatureFlag := false
-			for _, featureFlag := range velero.Spec.VeleroFeatureFlags {
-				if featureFlag == enableCSIFeatureFlag {
-					foundCSIFeatureFlag = true
-					break
-				}
-			}
-			if !foundCSIFeatureFlag { // Not Found so append to feature flag
-				velero.Spec.VeleroFeatureFlags = append(velero.Spec.VeleroFeatureFlags, enableCSIFeatureFlag)
-			}
+			velero.Spec.VeleroFeatureFlags = append(velero.Spec.VeleroFeatureFlags, enableCSIFeatureFlag)
```

Comment:
Sorry if I'm missing something, if I set `enableCSIFeatureFlag` on the spec, then won't this add it twice if the plugin is found no matter what?

Reply:
Oh ignore, I see `removeDuplicateValues` now